### DISCUSSION TITLE
1974-03-30-madurodam-miniature-city/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#15] [Photographs]
# 1974-03-30-madurodam-miniature-city/

Lena at the Madurodam Miniature City

https://fanzoflenazavaroni.github.io/1974-03-30-madurodam-miniature-city/

**Comment by StephenMolloy1** [comment]
It's difficult to pin down a correct date for these photographs, I'm not brilliant with foreign language newspapers, but this 8th April paper is the earliest record that I can find of her in Holland.
https://krantenbankzeeland.nl/issue/pzc/1974-04-08/edition/0/page/7?query=%20Lena%20Zavaroni&sort=relevance

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for the link.

Who knows‽ first date I had was from The Press Association that gives a date of 25 April 1974.
Then I got the 30 March 1974 via  the [Nationaal Archief](https://www.nationaalarchief.nl/onderzoeken/zoeken?activeTab=photos&rm=list&searchTerm=lena%20zavaroni).
While the link you sent can confirm Lena was in Holland on the 8th April 1974 the article does not mention when Lena first went there or when she left so the date of the photos from Madurodam Miniature City is still open to question for now I feel I have to leave the post on this date as it's the one currently provide by Nationaal Archief and as it the county's national archives I would hope they got the date right but as we both know that may not be the case, so I close this comment as I opened it Who knows‽

Here is a link to [Google Translate](https://translate.google.co.uk/?sl=auto&tl=en&text=AMSTELVEEN-%0A-%0AEen%20volwassen%0Aact%2C%20een%20bijna%20volwassen%20stem%2C%20maar%20een%20lijfje%20als%20een%20kind%20van%20zes%2C%20zeven%20jaar%3A%20dat%20zijn%20zo%20de%20verworvenheden%20van%20de%20tienjarige%20Engelse%20Lena%20Zava-%20roni.%20Een%20meisje%20waarvan%20de%20kenners%20zeggen%3A%2C%2CDat%20wordt%20een%20grote%20ster%3B%20ze%20is%20een%20geboren%20comedienne%22.%0A-%0ALena%20was%20even%20in%20ons%20land%20op%20de%20hielen%20gezeten%20door%20een%20meneer%20van%20de%20Engelse%20arbeidsinspectie%0Aom%0Ahier%20een%20zilveren%20plaat%20te%20krijgen%2C%20omdat%20er%20in%20een%20recordtijd%20300.000%20exemplaren%20van%20haar%20eerste%20plaatje%20zijn%20verkocht%3A%20Ma!%20He%27s%20making%20eyes%20at%20me!%22.%0A%0AIn%20Engeland%20zit%20haar%20plaatje%20boven%20in%20de%20top-tien%20en%20alsof%20het%20allemaal%20niks%20is%2C%20heeft%20Lena%20nu%20ook%20al%20een%20elpee%20opgenomen%2C%20die%20overigens%20dezelf-%20de%20titel%20als%20het%20singletje%20heeft%20gekre%20gen.%20Lena%20is%20geboren%20in%20Schotland.%20Ma%20is%20een%20verdienstelijke%20amateur-zange-%20res%2C%20pa%20een%20professionele%20muzikant.%20Haar%20Italiaanse%20naam%20komt%20van%20de%20vader%20van%20pa.%0A%0AEen%20natuurtalent%2C%20zeggen%20ze%20in%20En-%20geland%20van%20Lena%20en%20daar%20kunnen%20ze%20wel%20eens%20gelijk%20in%20hebben.%20Ze%20heeft%20en%20dat%20is%20toch%20wel%20opvallend%20voor%20zo%27n%20klein%20ding%20een%20opvallend%20krachtige%20stem%20en%20een%20goede%20mimiek.%20Ze%20wordt%20thans%2C%20voor%20zover%20de%20schooluren%20het%20toelaten%2C%20muzikaal%20opgefokt%20door%20Phi-%20lip%20Solomon%2C%20de%20man%20die%20al%20eerder%20piepjong%20talent%20onder%20handen%20had%2C%20zoals%20Neil%20Reid.%0A%0AVan%20het%20populaire%20Engelse%20teevee-%20programma%20Opportunity%20Knocks%22%20heeft%20Lena%20inmiddels%20een%20prijs%20gekre%20gen.%20Hier%20is%20Lena%20via%20de%20radio%20al%20wel%20te%20horen%2C%20maar%20nog%20niet%20via%20de%20teevee%20te%20zien.%20Van%20de%20arbeidsinspectie%20mag%20ze%20maar%20veertig%20dagen%20per%20jaar%20het%20land%20uit%2C%20als%20het%20om%20zingen%20gaat.%20Nog%20even%20geduld%20hebben%20dus.&op=translate) for the article on Lena that you sent me a link for today.

I got it translated by taking a screenshot of related text and then used [imagetotext](https://www.imagetotext.info) to get the text then copied and pasted that into Google Translate at some stage I will try and sort out the errors that are always made during Optical Character Recognition (OCR).

**Reply by StephenMolloy1** [reply]
Found three Dutch newspapers for 1st April with photographs of Lena at Madurodam so 30th March is probably the correct date.
https://www.delpher.nl/nl/kranten/view?query=%22Zavaroni%22&coll=ddd&sortfield=date&identifier=KBPERS01:002900001:mpeg21:a00004&resultsidentifier=KBPERS01:002900001:mpeg21:a00004&rowid=2

https://www.delpher.nl/nl/kranten/view?query=%22Zavaroni%22&coll=ddd&sortfield=date&identifier=ddd:010958425:mpeg21:a0145&resultsidentifier=ddd:010958425:mpeg21:a0145&rowid=4

https://www.delpher.nl/nl/kranten/view?query=%22Zavaroni%22&coll=ddd&sortfield=date&identifier=ddd:010375205:mpeg21:a0033&resultsidentifier=ddd:010375205:mpeg21:a0033&rowid=5

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for the links.

I added a section to the post about the [Dates](https://fanzoflenazavaroni.github.io/1974-03-30-madurodam-miniature-city/#dates) and a new post as an example to show how there may be more to a [typo](https://fanzoflenazavaroni.github.io/2012-05-30-the-daily-record/) than it may at first seem and why we should never take anything at face value.

**Reply by StephenMolloy1** [reply]
I notice that Dorothy Solomon mentioned in one of the articles that Lena wasn't allowed in a television studio after 5:00pm. This must mean that she recorded her Top Of The Pops appearance in the afternoon.

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for pointing that out to me, that reinforces Lena's first appearance on TOTP which is available on YouTube as the actual broadcast and not as I thought it was possibly a camera rehearsal. I must amend my post for Lena's first appearance on TOTP.
